pkg/schema/tracker.go

```diff
@@ -92,6 +95,33 @@ func (tr *Tracker) GetTable(db, table string) (*model.TableInfo, error) {
 	return t.Meta(), nil
 }
 
+// GetCreateTable returns the `CREATE TABLE` statement of the table.
+func (tr *Tracker) GetCreateTable(ctx context.Context, db, table string) (string, error) {
+	name := dbutil.TableName(db, table)
+	// use `SHOW CREATE TABLE` now, another method maybe `executor.ConstructResultOfShowCreateTable`.
+	rs, err := tr.se.Execute(ctx, fmt.Sprintf("SHOW CREATE TABLE %s", name))
+	if err != nil {
+		return "", err
+	}
+	defer rs[0].Close()
```

Comment:
do we need to judge the `len(rs)`?

Reply:
added in https://github.com/pingcap/dm/pull/764/commits/0a13645341f4e2da90f015feb01b01cd3ded1f79.